N24DCCN058/Baitapbuoi5/Bai10.cpp

```diff
@@ -0,0 +1,17 @@
+#include <iostream>
+
+using namespace std;
+int main() {
+    int n , count = 0 ;
+    cin >> n ; 
+    for(int i = 2 ; i < n ; i++){
```

Comment:
Đối với vòng for, bản chất là vòng lặp được sử dụng trong trường hợp biết trước số lần lặp, Nhưng ở đây biến n lại thay đổi sau mỗi lần lặp. Em cân nhắn sài vòng while để vừa đúng bản chất vừa tránh lỗi vặt nha (đối với vòng while trong bài này sẽ có vài chổ xử lý tổi ưu được). Ý tưởng đúng rồi

Reply:
oke anh